pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -704,8 +704,15 @@ public ApnsPayloadBuilder setUrlArguments(final String... arguments) {
      * @param value the value of the custom property
      *
      * @return a reference to this payload builder
+     *
+     * @throws IllegalArgumentException if the key equals to "aps"
+     *
+     * @since 0.15.2
```

Comment:
I'm torn on this; this is technically a breaking change, but this method has been around much longer. I'm inclined to leave out the `@since` tag in this case, but am open to other ideas!

Reply:
Ok, I've removed the `@since` tag 👀

Reply:
I'll make sure to call it out in the release notes, but agree that isn't entirely ideal. We're getting close to 1.0, though, and I'm planning on getting much stricter about semantic versioning when we get there.

Reply:
Thanks, it'll be great! 😍